CHANGELOG.md

```diff
@@ -402,6 +404,7 @@
 [@leffelmania]: https://github.com/LeffelMania
 [@simpleapp]: https://github.com/SimpleApp
 [@tsvetelinvladimirov]: https://github.com/TsvetelinVladimirov
+[#1029]: https://github.com/DataDog/dd-sdk-ios/issues/1029
```

Comment:
Lets move this line up so we keep grouping issues vs. contributors 👍

Reply:
Done 👍.

I'm using [`pimpmychangelog`](https://github.com/pcreux/pimpmychangelog) for formatting and it was inserted there automatically. However, due to manual adjustments others do it requires more and more manual patches on my side. Would be nice to streamline CHANGELOG formatting.

Reply:
agreed!